classes/game.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './item'
+require 'securerandom'
+require 'date'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at, :publish_date
+  attr_reader :id
```

Comment:
Kinldy remove the atributes that already exist in the parent class. Take a look at Item class, and the ones we have there, you don't need to repeat here!